packages/storefront/src/pages/patterns/forms/example-create-account.vue

```diff
@@ -14,53 +14,48 @@
         <form novalidate @submit.prevent="onSubmit">
           <p-flex direction="{ base: 'column', m: 'row' }" class="form-grid-item-container">
             <p-flex-item width="{base: 'full', m: 'one-third'}" class="form-grid-item">
-              <p-select-wrapper
+              <p-select
                 :theme="storefrontTheme"
                 label="Salutation"
-                v-bind:message="bag.errors.salutation"
-                v-bind:state="getState('salutation')"
+                :message="bag.errors.salutation"
+                :state="getState('salutation')"
+                @update="onCustomSelectUpdate"
+                @blur="onFieldBlur"
+                :ref="validateFieldName('salutation')"
+                :name="validateFieldName('salutation')"
+                required
               >
-                <select
-                  :ref="validateFieldName('salutation')"
-                  v-bind:name="validateFieldName('salutation')"
-                  v-model="bag.data.salutation"
-                  v-on:change="onFieldBlur"
-                  required
-                >
-                  <option value hidden></option>
-                  <option value="Mr.">Mr.</option>
-                  <option value="Mrs.">Mrs.</option>
-                </select>
-              </p-select-wrapper>
+                <p-select-option value hidden></p-select-option>
```

Comment:
```suggestion
```
If we don't want to allow deselection is this case we can omit the option without value